gameplay-controller.js

```diff
@@ -35,11 +33,13 @@ export default class GameplayController {
             kills: 0,
             milestone: 0,
             weapons: [0],
+            score: 0,
         };
```

Comment:
**issue**: The use of a hardcoded random value (`const rand = 1;`) may not provide the intended randomness. Consider using `Math.random()` for true randomness.